.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "cvm-runtime"]
+	path = cvm-runtime
+	url = https://github.com/CortexFoundation/cvm-runtime.git
+	branch = master
```

Comment:
Use a stable branch name here like v1,v2 instead of master

Reply:
got it